go/vt/mysqlctl/rice_box_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mysqlctl
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	rice "github.com/GeertJohan/go.rice"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	mysqlCtlRiceBoxLocation string = "../../../config"
```

Comment:
nit: should be `mySqlCtl` or `mysqlctl`. I prefer the second option.
EDIT: Or even `mySQLCtl`?? 🤷